content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -282,9 +282,9 @@ define readiness distinct from completion. This is enforced during validation.
 
 Use `activeDeadlineSeconds` on the Pod to prevent init containers from failing forever.
 The active deadline includes init containers.
-However it is recommended to use `activeDeadlineSeconds` if user deploy their application
-as a Job, because `activeDeadlineSeconds` has an effect even after initContainer finished.
-The Pod which is already running correctly would be killed by `activeDeadlineSeconds` if you set.
+However it is NOT recommended to use `activeDeadlineSeconds` if the user deploys their application
+as a Job, because `activeDeadlineSeconds` has an effect even after init containers have finished.
+The Pod which is already running correctly would be killed by `activeDeadlineSeconds` if you set it.
```

Comment:
```suggestion
Do not use `activeDeadlineSeconds` if you deploy your application as a Job, because `activeDeadlineSeconds` has an effect even after init containers have finished. Setting `activeDeadlineSeconds` would terminate correctly  running Pods.
```

Some optional related cleanup in this paragraph. LGTM though.